analyzers/src/SonarAnalyzer.CFG/SonarAnalyzer.CFG.csproj

```diff
@@ -13,10 +13,14 @@
       <!-- This package is a dependency of Microsoft.CodeAnalysis.CSharp.Workspaces. It is safe to use since it's compatible with .Net Portable runtime -->
       <NoWarn>NU1701</NoWarn>
     </PackageReference>
+    <PackageReference Include="System.Collections" Version="4.3.0" />
```

Comment:
This changes transitive 4.3.0 to explicit 4.3.0 if I compare `dotnet list package --include-transitive`. Does it really have to be here?

Reply:
It was a misunderstanding, there were a few problems with other libraries since a downgrade was detected at build time, and I thought that this library was also part of the problem:
![image](https://user-images.githubusercontent.com/56015273/142592574-9a84c4ec-2586-48f6-820f-072e1eaf9499.png)